massdriver.yaml

```diff
@@ -0,0 +1,48 @@
+schema: draft-07
+name: "aws-vpc-peering-connection"
+description: "An AWS VPC Peering Connection is a networking connection between two VPCs that enables you to route traffic between them privately."
+source_url: github.com/massdriver-cloud/aws-vpc-peering-connection
+access: private
+type: "infrastructure"
+
+
+params:
+  properties:
+    accepter_vpc_arn:
+      type: string
+      title: Remote VPC ARN
+      description: "**IMPORTANT: Only set this value if you haven't connected a remote \"accepter\" VPC to the bundle!!!**\n If an accepter VPC is connected, this field is ignored and the value will be extracted from the connection artifact. Use this field if the remote VPC isn't managed by Massdriver or exists in different AWS account than the requester VPC. **This will require you to [accept the peering connection](https://docs.aws.amazon.com/vpc/latest/peering/accept-vpc-peering-connection.html) and [update the route tables](https://docs.aws.amazon.com/vpc/latest/peering/vpc-peering-routing.html) of the accepter VPC manually**!"
+      $md.immutable: true
+      $ref: https://raw.githubusercontent.com/massdriver-cloud/artifact-definitions/main/definitions/types/aws-arn.json
+    accepter_vpc_cidr:
+      type: string
+      title: Remote VPC CIDR
+      description: "**IMPORTANT: Only set this value if you haven't connected a remote \"accepter\" VPC to the bundle!!!**\n If an accepter VPC is connected, this field is ignored and the value will be extracted from the connection artifact. Use this field if the remote VPC isn't managed by Massdriver or exists in different AWS account than the requester VPC. **This will require you to [accept the peering connection](https://docs.aws.amazon.com/vpc/latest/peering/accept-vpc-peering-connection.html) and [update the route tables](https://docs.aws.amazon.com/vpc/latest/peering/vpc-peering-routing.html) of the accepter VPC manually**!"
+      $md.immutable: true
+      pattern: ^(?:10\.(?:[0-9]|[0-9]{2}|1[0-9][0-9]|2[0-4][0-9]|25[0-5])|172\.(?:1[6-9]|2[0-9]|3[0-1])|192\.168)(?:\.(?:[0-9]|[0-9]{2}|1[0-9][0-9]|2[0-4][0-9]|25[0-5])){2}(?:/(?:1[6-9]|2[0-4]))$
```

Comment:
Could we use the autocidr feature for this to make a drop-down?

Reply:
This field isn't for creating a VPC, its for connecting to an existing VPC. This CIDR must match the CIDR of an existing, external DB.